test/addons/layout-change/lib/test-cuddlefish-loader.js

```diff
@@ -21,11 +21,6 @@ exports["test compatibility"] = function(assert) {
   assert.equal(require("tabs"),
                require("sdk/tabs"), "sdk/tabs -> tabs");
 
-  if (app.is("Firefox")) {
-    assert.equal(require("widget"),
-                 require("sdk/widget"), "sdk/widget -> widget");
-  }
```

Comment:
keeping the "widget" -> "sdk/widget" mapping would break a node_module named widget, and since we don't need it any longer we should remove.  We shouldn't keep code around that doesn't do anything right?